core/src/main/codegen/templates/Parser.jj

```diff
@@ -1666,8 +1666,8 @@ SqlNode SqlDelete() :
 SqlNode SqlUpdate() :
 {
     SqlNode table;
-    SqlNode secondTable = null;
-    SqlIdentifier secondAlias = null;
+    SqlNodeList sourceTables = null;
```

Comment:
How is this approach compares with having sourceTables initialized as an empty SqlNodeList with the final keyword? Same question for the sourceAliases.

Reply:
I don't think it matters much as the only thing that changes is which condition you check for when unparsing, either list != null or list.size() > 0. I suppose making it an empty list is better as it avoids null but it seems like a marginal difference.

Reply:
Yeah, it would be up to you